migrations/versions/eb9f8639d610_add_tokens.py

```diff
@@ -0,0 +1,33 @@
+"""Add tokens
+
+Revision ID: eb9f8639d610
+Revises: 
+Create Date: 2019-12-09 14:38:26.434966
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'eb9f8639d610'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('token',
```

Comment:
can you run lint in here?
Should we run black on those autogenerated code snippets?

Reply:
Done.